src/main/java/core/basesyntax/strategy/operation/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void performOperation(FruitTransaction transaction) {
+        Storage.storage.put(transaction.getFruitName(), transaction.getQuantity());

```

Comment:
It's good practice to check if the quantity is not negative before updating the storage. Even though the 'balance' operation might assume that the data is valid, ensuring non-negative quantities can prevent potential issues if the operation is used incorrectly.